extend/component/code-patterns/interface.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Code Pattern Interface
+permalink: /extend/component/code-patterns/interface
+---
+
+* TOC
+{:toc}
+
+In the [Overview](/extend/component/code-patterns/overview) you could see the code patterns from the user's point of view.
+This page describes how the code patterns works internally, as a part of Keboola Connection.
+
+## Common Interface
+
+Code pattern is a special type of the [Component](/extend/component/), therefore, common interface applies to it. 
+In the links below you can find the basic information to integrate your own components into Keboola Conneciton.
+- [Component Common Interface](/extend/common-interface/) 

```

Comment:
toto bych vynechal mozna? a hlavne napsal, ze to nema run, ale jen jednu synchronni akci

Reply:
no koukam, ze to tam pak je potom, trosku bych to prehazel :)

Reply:
ten run tam pridam, .... tie odkazy by som tam bezne nedaval, ale vraj to ma byt vsetko pokope pre vyvojara co nikdy s keboola nic nerobil, tak neviem, comu dat teraz prednost.

Reply:
jj chapu, nech to klidne zatim tak, je to sice nesmysl, ale nechce se mi se o tom ted handrkovat

spravnej postup ma byt: projdes tutorial -> prosel jsi -> jdes delat co te zajima (treba code pattern); jinak to nejde popsat

Reply:
https://keboola.atlassian.net/browse/DOCS-130